### DISCUSSION TITLE
FMOD generator

### DISCUSSION THREAD

**Discussion by francescoameglio** [Wasted-Audio/hvcc#195] [Ideas]
I was wondering if it would be possible to build a generator for FMOD... Wwise is slowly getting less attraction compared to FMOD, especially in the smaller game productions. It would be awesome to turn c/cpp into a FMOD plugin.

**Comment by dromer** [comment]
Of course it's possible. But someone would have to do the work.

**Reply by francescoameglio** [reply]
Good point :) If only I had more knowledge in coding I would definitely help on this but my expertise stops in PD/MaxMsp programming.

**Reply by dromer** [reply]
I created a feature request from the discussion -> https://github.com/Wasted-Audio/hvcc/issues/196

**Reply by francescoameglio** [reply]
Thank you so much @dromer
